package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-data-grid",
-  "version": "0.14.46",
+  "version": "0.15.0",
```

Comment:
After discussion We agreed that version should bump to 1.0.0 and not 0.15.0, as we will not be following the same version as react anymore it wouldn't make much sense to keep this kind of versioning.